src/Servers/IIS/IIS/perf/Microbenchmarks/Properties/launchSettings.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Do you know how to prevent this file from being created? I think it breaks the microbenchmarks (although they are already broken anyways)

Reply:
Not sure, we can look into it as a follow up.